templates/keystoneapi/config/keystone-api-config.json

```diff
@@ -2,21 +2,27 @@
     "command": "/usr/sbin/httpd",
     "config_files": [
         {
-            "source": "/var/lib/emptydir/keystone.conf",
+            "source": "/var/lib/config-data/merged/keystone.conf",
             "dest": "/etc/keystone/keystone.conf",
             "owner": "keystone",
             "perm": "0600"
         },
         {
-            "source": "/var/lib/config-data/httpd.conf",
+            "source": "/var/lib/config-data/merged/custom.conf",
+            "dest": "/etc/keystone/keystone.conf.d/custom.conf",
+            "owner": "keystone",
+            "perm": "0600"
+        },
+        {
+            "source": "/var/lib/config-data/merged/httpd.conf",
             "dest": "/etc/httpd/conf/httpd.conf",
             "owner": "root",
```

Comment:
this is presumable because apache is running as root? can we avoid that somehow.

Reply:
The container itself isn't being granted privs on the cluster so it should be fine right? This has more to do with the defaults of Apache service itself so we should probable stick with root I think to avoid other issues.